perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -1432,4 +1432,10 @@ sub report_new_munge_before_insert {
     }
 }
 
+sub report_on_private_contacts {
+    my ($self, $category) = @_;
+
+    return [ FixMyStreet::DB->resultset('Contact')->search({category => $category})->all ];
```

Comment:
I guess this would also return deleted, which I guess might not be wanted, happy to keep the inactive in here, or exclude deleted (there's a `not_deleted` you could call before the search)